artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -4484,6 +4484,10 @@ public Integer expectedHits() {
        * @throws Exception
        */
       public abstract boolean actMessage(Transaction tx, MessageReference ref) throws Exception;
+
+      public boolean reachExpectedHits(int currentHits) {
```

Comment:
I would name it 'reachedExpectedHits' or my personal preference 'expectedHitsReached' to make the meaning more clear

Reply:
Yeah I think expectedHitsReached is more consistent one. I'll update. Thanks Tim.